mods/cnc/maps/funpark01/map.yaml

```diff
@@ -428,6 +428,16 @@ Actors:
 Smudges:
 
 Rules:
+	^Palettes:
+		-PlayerColorPalette:
+		IndexedPlayerPalette:
+			BasePalette: terrain
+			RemapIndex: 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191
+			PlayerIndex:
+				Nod: 161, 200, 201, 202, 204, 205, 206, 12, 201, 202, 203, 204, 205, 115, 198, 114
+				Neutral: 192, 164, 132, 155, 133, 197, 112, 12, 163, 132, 155, 133, 134, 197, 154, 198
+				Civilian: 192, 164, 132, 155, 133, 197, 112, 12, 163, 132, 155, 133, 134, 197, 154, 198
+				Dinosaur: 192, 164, 132, 155, 133, 197, 112, 12, 163, 132, 155, 133, 134, 197, 154, 198
```

Comment:
`Civilian` and `Dinosaur` can be added here to extend `^PlayerPalettes` or placed directly in `^PlayerPalettes`.